src/extra/gpkg.jl

```diff
@@ -0,0 +1,215 @@
+# Reminder: We have to set this (on-write) after there's some content
+# so the database file is not zero length
+
+const GP10_APPLICATION_ID = 74777363 #0x47503130
+const GP11_APPLICATION_ID = 119643780 # 0x47503131
+const GPKG_APPLICATION_ID = 1196444487 # 0x47504B47
+const GPKG_1_2_VERSION = 10200
+const GPKG_1_3_VERSION = 10300
+const GPKG_1_4_VERSION = 10400
+
+
+const CREATE_GPKG_GEOMETRY_COLUMNS =
+    "CREATE TABLE gpkg_geometry_columns ("*
+    "table_name TEXT NOT NULL,"*
+    "column_name TEXT NOT NULL,"*
+    "geometry_type_name TEXT NOT NULL,"*
+    "srs_id INTEGER NOT NULL,"*
+    "z TINYINT NOT NULL,"*
+    "m TINYINT NOT NULL,"*
+    "CONSTRAINT pk_geom_cols PRIMARY KEY (table_name, column_name),"*
+    "CONSTRAINT uk_gc_table_name UNIQUE (table_name),"*
+    "CONSTRAINT fk_gc_tn FOREIGN KEY (table_name) REFERENCES "*
+    "gpkg_contents(table_name),"*
+    "CONSTRAINT fk_gc_srs FOREIGN KEY (srs_id) REFERENCES gpkg_spatial_ref_sys "*
+    "(srs_id)"*
+    ")"
+
+const CREATE_GPKG_REQUIRED_METADATA =  "CREATE TABLE gpkg_spatial_ref_sys ("*
+                    "srs_name TEXT NOT NULL,"*
+                    "srs_id INTEGER NOT NULL PRIMARY KEY,"*
+                    "organization TEXT NOT NULL,"*
+                    "organization_coordsys_id INTEGER NOT NULL,"*
+                    "definition  TEXT NOT NULL,"*
+                    "description TEXT,"*
+                    "definition_12_063 TEXT NOT NULL,"*
+                    "epoch DOUBLE"*
+                ")"*
+                ";"*
+                "INSERT INTO gpkg_spatial_ref_sys ("*
+                "srs_name, srs_id, organization, organization_coordsys_id, "*
+                "definition, description, definition_12_063"*
+                ") VALUE ("*
+
+                "'WGS 84 geodetic', 4326, 'EPSG', 4326, '"*
+                "GEOGCS[\"WGS 84\",DATUM[\"WGS_1984\",SPHEROID[\"WGS "*
+                "84\",6378137,298.257223563,AUTHORITY[\"EPSG\",\"7030\"]],"*
+                "AUTHORITY[\"EPSG\",\"6326\"]],PRIMEM[\"Greenwich\",0,AUTHORITY["*
+                "\"EPSG\",\"8901\"]],UNIT[\"degree\",0.0174532925199433,AUTHORITY["*
+                "\"EPSG\",\"9122\"]],AXIS[\"Latitude\",NORTH],AXIS[\"Longitude\","*
+                "EAST],AUTHORITY[\"EPSG\",\"4326\"]]"*
+                "', 'longitude/latitude coordinates in decimal degrees on the WGS "*
+                "84 spheroid', "*
+
+                "'GEODCRS[\"WGS 84\", DATUM[\"World Geodetic System 1984\", "*
+                "ELLIPSOID[\"WGS 84\",6378137, 298.257223563, "*
+                "LENGTHUNIT[\"metre\", 1.0]]], PRIMEM[\"Greenwich\", 0.0, "*
+                "ANGLEUNIT[\"degree\",0.0174532925199433]], CS[ellipsoidal, "*
+                "2], AXIS[\"latitude\", north, ORDER[1]], AXIS[\"longitude\", "*
+                "east, ORDER[2]], ANGLEUNIT[\"degree\", 0.0174532925199433], "*
+                "ID[\"EPSG\", 4326]]'"*
+                ")"*
```

Comment:
Can you please explain why we have EPSG:4326 hard-coded into this string? Our goal is to extract the WKT string with `CoordRefSystems.wkt(crs)` and paste it in the file during `GeoIO.save`.